basic_flutter/lib/presentation/widgets/employee_bloc_builder.dart

```diff
@@ -1,66 +1,65 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 
-import '/repository/data_provider.dart';
 import '../widgets/list_item.dart';
 import '../../business_logic/cubits/response_cubit.dart';
 import '../../business_logic/cubits/response_state.dart';
 
 class EmployeeBlocBuilder extends StatelessWidget {
   const EmployeeBlocBuilder({
     Key? key,
-    required this.dataProvider,
   }) : super(key: key);
 
-  final DataProvider dataProvider;
   // use cubit dataprovider
   // move deleteEmployeeEntry to ResponseCubit
```

Comment:
old comments?

Reply:
?

Reply:
yes, old comments, I've removed them in other PRs and will do in this one as well.